lib/datastore.rb

```diff
@@ -0,0 +1,27 @@
+class Datastore
+
+    def initialize(data={})
+        @data = data
+    end
+
+    def clear
+        @data.clear
+    end
+
+    def has_key?(key)
+        @data.has_key?(key)
+    end
+
+    def store(key, value)
+        @data.store(key, value)
+    end
+
+    def load(game_id, key)
+        @data[game_id][key]
```

Comment:
Here's a Ruby method that might be of interest to you: [Array#dig](https://apidock.com/ruby/Array/dig).